examples/menubar/images/aria-current.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think you can delete this file, now that you have moved the svg definition into the css! :)